spec/web_helpers.rb

```diff
@@ -0,0 +1,23 @@
+def sign_up(name: 'Andrew Htun', username: 'htunny', email: 'andy_htun@hotmail.com', password: 'Apples', password_confirmation: 'Apples')
```

Comment:
Metrics/LineLength: Line is too long. [137/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)